tests/unit/modules/network/vyos/test_vyos_user.py

```
@@ -59,7 +59,27 @@ def test_vyos_user_password(self):
         result = self.execute_module(changed=True)
         self.assertEqual(
             result["commands"],
-            ["set system login user ansible authentication plaintext-password test"],
+            ["set system login user ansible authentication plaintext-password 'test'"],
+        )
+
+    def test_vyos_user_password_special_chars(self):
+        set_module_args(dict(name="ansible", configured_password="test$123!@"))
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "configured_password"

(S106)